stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -41,7 +41,122 @@ module SwiftGlibc [system] {
   link "execinfo"
 % end
 
-  header "SwiftGlibc.h"
+% if CMAKE_SDK not in ["ANDROID"]:
+  header "stdc-predef.h"
+  header "features.h"
```

Comment:
All these headers were removed for Android because they don't exist in Bionic, except for `features.h`, [which is only in Bionic for glibc compatibility and is unnecessary](https://android.googlesource.com/platform/bionic/+/master/libc/include/features.h#33) plus it causes build issues in C++ Interop.